mpf/devices/segment_display/segment_display.py

```diff
@@ -145,7 +148,14 @@ def add_text_entry(self, text, color, flashing, flash_mask, transition, transiti
 
         This will replace texts with the same key.
         """
+
+        if len(color) == 0:
```

Comment:
This could be more readable (and do the same thing) with a generic truthiness check:
```
if not color:
  color = self._current_state.text.get_colors()
```